worldengine/drawing_functions.py

```diff
@@ -251,7 +209,7 @@ def _draw_glacier(pixels, x, y):
     pixels[y, x] = (rg, rg, 255, 255)
 
 
-def _draw_cold_parklands(pixels, x, y):
+def _draw_cold_parklands(pixels, x, y, w, h):
```

Comment:
I am wrong or w and h are not used here? Are they passed for consistency with other methods?

Reply:
Exactly. IIRC it was that way, then I removed them in the first round of refactoring, and now I put them back in so that duck typing works. cf. `_draw_biome`, this gets called as `_func`.

Could be solved with an ìf` but branching is slow.

Reply:
makes sense